packages/page-accounts/src/modals/Create.tsx

```diff
@@ -61,10 +61,17 @@ interface DeriveValidationOutput {
 const DEFAULT_PAIR_TYPE = 'sr25519';
 const STEPS_COUNT = 3;
 
+function isHDDerivePath (path: string) {
+  // TODO: add more validation
+  const firstLetter = path[0];
+
+  return firstLetter === 'm' || firstLetter === 'M' || firstLetter === "m'" || firstLetter === "M'";
+}
+
 function getSuri (seed: string, derivePath: string, pairType: PairType): string {
   return pairType === 'ed25519-ledger'
     ? u8aToHex(hdLedger(seed, derivePath).secretKey.slice(0, 32))
-    : `${seed}${derivePath}`;
+    : isHDDerivePath(derivePath) ? `${seed}/${derivePath}` : `${seed}${derivePath}`;
```

Comment:
This only needs to apply to the specific ethereum-like type. Not to all. So in the case where Ethereum is detected (assuming the path is validated with the `hd` add it via `#{seed}/${path}` (All the rest left as-is)